config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root to: "pages#home"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :animals
```

Comment:
Lets specify the routes that we need, this resources command give all the 7 actions, we don't need all of them